docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -62,26 +55,19 @@ When you [Establish a Tax Provider connection](#establishing-a-connection), incl
   You must liaise with the merchant to provide them with customized URLs. BigCommerce merely consumes the URL values the merchant enters and uses the provided string value for Tax Provider API requests.
 </Callout>
 
-### Sandbox tax provider configuration
-
-Sandbox tax provider configurations are nested within your production tax provider configuration, as such tax providers requiring a sandbox tax provider configuration should also provide details for their production tax provider configuration.
+### Test tax provider (optional)
 
-Additionally, when providing the details for a sandbox tax provider configuration, it's unnecessary to specify the below properties because they are inherited from the primary production tax provider configuration (all other properties are still required):
+You can build a tax provider instance that supports test transactions. This test tax provider instance will sit alongside a primary production tax provider instance, servicing merchant-facing use cases or supporting the tax provider's own development processes.
 
-* Merchant support email
-* Merchant support URL
-* Tax provider visibility
-* Platform availability, i.e. all stores or only specified store hashes
-* Supported / unsupported countries
+Tax provider instances that support test transactions have lower expectations than those supporting production traffic (e.g. timeouts, error handling), and so it's important to differentiate between a production tax provider and a test tax provider when sharing provider details with BigCommerce.
```

Comment:
```suggestion
When you [share provider details](#sharing-provider-details-with-bigcommerce), it's important to specify the transaction type for your tax provider instance. Tax provider instances that support test transactions have lower expectations than those supporting production traffic like timeouts or error handling.
```

Reply:
Great, I had also considered adding a link back to the sharing provider details section - cheers!